apps/services/user-notification/src/app/modules/notifications/dto/createNotification.dto.ts

```diff
@@ -1,17 +1,12 @@
-import { IsEnum, IsNotEmpty, IsString } from 'class-validator'
+import { IsEnum, IsOptional, IsString } from 'class-validator'
 import { ApiProperty } from '@nestjs/swagger'
 import { IsNationalId } from '@island.is/nest/validators'
 import { MessageTypes } from '../types'
 
-export class TypeValidator {
+export class CreateNotificationDto {
   @IsEnum(MessageTypes)
-  @ApiProperty({ enum: MessageTypes })
+  @ApiProperty({ example: MessageTypes.NewDocumentMessage })
```

Comment:
If this project is being used for OpenAPI generation then the yml schema may change in the schema section upon removal of the enum property.

Are you sure there are no schema changes?